ferrocene/doc/qualification-plan/src/standards/iso-26262.rst

```diff
@@ -12,207 +12,259 @@ The following are sections in |iso_ref| Part 8.
      - :doc:`Evaluation Report - rustfmt - Use cases <evaluation-report:rustfmt/use-cases>`
+     - :doc:`Evaluation Report - ferrocene-self-test - Use cases <evaluation-report:self-test/use-cases>`
 
    * - 11.4.5.2
      - :doc:`Evaluation Report - rustc - Tool analysis <evaluation-report:rustc/tool-analysis>`
      - :doc:`Evaluation Report - rustfmt - Tool analysis <evaluation-report:rustfmt/tool-analysis>`
+     - :doc:`Evaluation Report - ferrocene-self-test - Tool analysis <evaluation-report:self-test/tool-analysis>`
 
    * - 11.4.5.2.a
      - See super section above
      - See super section above
+     - See super section above
 
    * - 11.4.5.2.a.1
      - See super section above
      - See super section above
+     - See super section above
 
    * - 11.4.5.2.a.2
      - See super section above
      - See super section above
+     - See super section above
 
    * - 11.4.5.2.b
      - See super section above
      - See super section above
+     - See super section above
 
    * - 11.4.5.2.b.1
      - See super section above
      - See super section above
+     - See super section above
 
    * - 11.4.5.2.b.2
      - See super section above
      - See super section above
+     - See super section above
 
    * - 11.4.5.2.b.3
      - See super section above
      - See super section above
+     - See super section above
 
    * - 11.4.5.3
      - :doc:`Evaluation Report - rustc - Tool analysis <evaluation-report:rustc/tool-analysis>`
      - :doc:`Evaluation Report - rustfmt - Tool analysis <evaluation-report:rustfmt/tool-analysis>`
+     - :doc:`Evaluation Report - ferrocene-self-test - Tool analysis <evaluation-report:self-test/tool-analysis>`
 
    * - 11.4.5.4
      - :doc:`Evaluation Report - rustc - Tool analysis <evaluation-report:rustc/tool-analysis>`
      - :doc:`Evaluation Report - rustfmt - Tool analysis <evaluation-report:rustfmt/tool-analysis>`
+     - :doc:`Evaluation Report - ferrocene-self-test - Tool analysis <evaluation-report:self-test/tool-analysis>`
 
    * - 11.4.6.1
      - :doc:`Evaluation Report - rustc - Qualification method <evaluation-report:rustc/method>`
      - N/A
+     - :doc:`Evaluation Report - ferrocene-self-test - Qualification method <evaluation-report:self-test/method>`
 
    * - 11.4.6.2
      - See sub items below
      - See sub items below
+     - See sub items below
 
    * - 11.4.6.2.a
      - :doc:`Safety Manual - Qualification Scope <safety-manual:scope>`
      - See ``rustc`` column
+     - See ``rustc`` column
 
    * - 11.4.6.2.b
      - :doc:`Qualification Report - Acceptability statement <qualification-report:statement>`
      - :doc:`Evaluation Report - rustfmt - Tool analysis <evaluation-report:rustfmt/tool-analysis>`
+     - :doc:`Evaluation Report - ferrocene-self-test - Tool analysis <evaluation-report:self-test/tool-analysis>`
 
    * - 11.4.6.2.c
      - :doc:`Qualification Report - Acceptability Statement <qualification-report:statement>`
      - N/A
+     - N/A
```

Comment:
We should add an acceptability statement for ferrocene-self-test, or (better!) reformulate the current one to fit to all tools.